docs/guides/auth/elevated-permissions.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Elevated Permissions
+sidebarTitle: Elevated Permissions
+description: Require extra permissions to perform critical operations
+icon: unlock
+---
+
+In some scenarios you may want to add an extra layer of security to perform certain actions or view certain data. For instance, you may wish to allow users to view their profile information after authenticating but you may want to require users to confirm changes to their profile by performing an extra validation step with a [security key](sign-in-webauthn).
+
+![overview-flow](/images/guides/auth/elevated-permissions/overview.png)
+
+This is accomplished by adding the claim `x-hasura-auth-elevated: $user-id` to the access token in response to the extra security challenge. With this new claim in mind you can start writing permissions that require this extra step.
+
+It is important to keep in mind the claim is added to the access token so, for as long as you have that access token, you will have elevated access. Once the access token is renewed (due to expiration or any other reason), the claim will be lost and a new security challenge will be required to add it back.
+
+# API Endpoints and SDK components
+
+To implement this functionality [auth](/product/authentication) [0.26.0](https://github.com/nhost/hasura-auth/releases/tag/0.26.0) introduces the new endpoint [/elevate/webauthn](/reference/auth/elevate-webauthn). This endpoint works the same way as [/signin/webauthn](/reference/auth/sign-in-using-email-via-fido2-webauthn-authentication) but it requires an Authorization header with a valid token to add the elevated claim to.
+
+In addition, a few methods and components have also been added to simplify it's usage:
+
+- [nhost.auth.elevateEmailSecurityKey](/reference/javascript/auth/elevate-email-security-key)
+- [React - useElevateSecurityKeyEmail](/reference/react/use-elevate-security-key-email)
+- [Next.js - useElevateSecurityKeyEmail](/reference/nextjs/use-elevate-security-key-email)
+- [Vue - useElevateSecurityKeyEmail](/reference/vue/use-elevate-security-key-email)
+
+# Protecting Hasura data
+
+You can use the claim `x-hasura-auth-elevated` in exactly the same way you would normally use `X-Hasura-User-Id` for added security. For instance, the following permissions would allow users to see their data without any extra secrity:
```

Comment:
```suggestion
You can use the claim `x-hasura-auth-elevated` in exactly the same way you would normally use `X-Hasura-User-Id` for added security. For instance, the following permissions would allow users to see their data without any extra security:
```